src/zzz_od/application/inventory_scan/pre_scan/pre_scan_app.py

```
@@ -0,0 +1,113 @@
+        """扫描代理人基础信息"""
+        log.debug("开始扫描代理人基础信息...")
+        time.sleep(1)# 等待代理人信息界面加载完成
+        #遍历所有代理人
+        max_iterations = 100  # 最大循环次数，避免无限循环
+        iteration = 0
+        agent_keys=[]
+        while iteration < max_iterations:
+            iteration += 1
+            screen = self.screenshot()
+            if screen is None:
+                log.error("截图失败，无法获取屏幕画面")
```

Ruff (0.15.7)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)